packages/pipeline-editor/src/SubmissionHandler.tsx

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2018-2020 IBM Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FrontendServices, RequestHandler } from '@elyra/application';
+import { showDialog, Dialog } from '@jupyterlab/apputils';
+import * as React from 'react';
+
+export class SubmissionHandler {
```

Comment:
Again, this looks good, but it would be good to actually have a facade service here as well, with something like PipelineService.submitPipeline()which will somewhat follow the pattern we are trying to establish throughout the project.

Reply:
I'll update this to be `PipelineService` I almost named it that during development anyway.

Reply:
renamed